test/snippet/search/configuration_modes_dynamic.cpp

```diff
@@ -0,0 +1,19 @@
+#include <seqan3/search/configuration/all.hpp>
+
+int main()
+{
+    seqan3::search_cfg::mode my_mode{}; // Enables the dynamic configuration, defaults to seqan3::search_cfg::all.
+
+    if (true)
+        my_mode = seqan3::search_cfg::best; // The best mode is selected.
+    else
+        my_mode = seqan3::search_cfg::strata{2}; // The strata mode is selected.
```

Comment:
The `else` branch is never executed...?

Reply:
yes, that is right. but that does not matter. It is a demo to show how to use it. And usually it would be by some condition you would do the one or the other thing. that is also more consistent with the text that describes this snippet.

Reply:
Okay, fine then 👍